pygluu/kubernetes/terminal/upgrade.py

```diff
@@ -35,4 +36,7 @@ def prompt_upgrade(self):
 
         image_names_and_tags = versions.get(self.settings.get("GLUU_UPGRADE_TARGET_VERSION"), {})
         self.settings.update(image_names_and_tags)
-        self.settings.store_data()
+
+        # reset this config to force image prompt
+        self.settings.set("EDIT_IMAGE_NAMES_TAGS", "")
```

Comment:
Although this is considered a stop point in CI/CD envs but the upgrade method nevertheless is not CI/CD and must be done by an administrator.